### PR TITLE
ROX-10813: plotted image vulnerabilities

### PR DESCRIPTION
**Pull request by keyallis**
## Description

PlottedImageVulnerabilities implementation to match the PlottedNodeVulnerabilities. Used in dashboards.

## Checklist
- [ ] Investigated and inspected CI test results

### CODE REVIEW

File: central/graphql/resolvers/plotted_image_vulnerabilities.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+// PlottedImageVulnerabilitiesResolver returns the data required by top risky images scatter-plot on vuln mgmt dashboard
+type PlottedImageVulnerabilitiesResolver struct {
+	root    *Resolver
+	all     []string
+	fixable int
+}
+
+func newPlottedImageVulnerabilitiesResolver(ctx context.Context, root *Resolver, args RawQuery) (*PlottedImageVulnerabilitiesResolver, error) {
+	query := withImageCveTypeFiltering(args.String())
+	allCveIds, fixableCount, err := getPlottedVulnsIdsAndFixableCount(ctx, root, RawQuery{Query: &query})
```
Comment (md2119) [inline central/graphql/resolvers/plotted_image_vulnerabilities.go:28]: Just a note @keyallis  and @charmik-redhat, the `getPlottedVulnsIdsAndFixableCount` uses the legacy datastore, which we will need to address. In postgres, new image vulns and node vulns should be used.

**Review by md2119 — APPROVED** [review]
A couple of comments, LG otherwise.

File: central/graphql/resolvers/image_components.go [hunk 2]
```diff
@@ -65,14 +66,13 @@ type ImageComponentResolver interface {
...
 	OperatingSystem(ctx context.Context) string
+	PlottedImageVulnerabilities(ctx context.Context, args RawQuery) (*PlottedImageVulnerabilitiesResolver, error)
 	Priority(ctx context.Context) int32
 	RiskScore(ctx context.Context) float64
 	Source(ctx context.Context) string
 	TopImageVulnerability(ctx context.Context) (ImageVulnerabilityResolver, error)
 	UnusedVarSink(ctx context.Context, args RawQuery) *int32
 	Version(ctx context.Context) string
-
-	PlottedVulns(ctx context.Context, args RawQuery) (*PlottedVulnerabilitiesResolver, error) // TODO
```
[the remark this answers is not in the document]
Comment (vjwilson) [reply central/graphql/resolvers/image_components.go:75]: @md2119 is correct. All the uses of `plottedVulns` are in this file on the FE, and **components** is not among the various versions that we call.

`ui/apps/platform/src/Containers/VulnMgmt/widgets/TopRiskyEntitiesByVulnerabilities.js`

**Pull request merged**